packages/labs/task/src/deep-equals.ts

```diff
@@ -4,121 +4,19 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-export const deepArrayEquals = <T extends ReadonlyArray<unknown>>(
-  oldArgs: T,
-  newArgs: T
-) =>
-  oldArgs === newArgs ||
-  (oldArgs.length === newArgs.length &&
-    oldArgs.every((v, i) => deepEquals(v, newArgs[i])));
-
-const objectValueOf = Object.prototype.valueOf;
-const objectToString = Object.prototype.toString;
-const {keys: objectKeys} = Object;
-const {isArray} = Array;
-
 /**
- * Recursively checks two objects for equality.
- *
- * This function handles the following cases:
- *  - Primitives: primitives compared with Object.is()
- *  - Objects: to be equal, two objects must:
- *    - have the same constructor
- *    - have same set of own property names
- *    - have each own property be deeply equal
- *  - Arrays, Maps, Sets, and RegExps
- *  - Objects with custom valueOf() (ex: Date)
- *  - Objects with custom toString() (ex: URL)
- *
- * Important: Objects must be free of cycles, otherwise this function will
- * run infinitely!
+ * You can import directly from @lit/task/deep-equals.js now.
```

Comment:
Does this deprecate imports of this file by being on the export? Or does that need to be in a `@fileoverview` or `@module` comment?

Reply:
I tried those techniques as well as a couple others, and none worked to deprecate the whole module